actionpack/test/controller/caching_test.rb

```diff
@@ -406,4 +419,25 @@ def test_caching_works_with_beginning_comment
     ActionView::PartialRenderer.expects(:collection_with_template).never
     get :index_with_comment
   end
+
+  def test_caching_works_with_collection_rendering_multiple_templates
+    get :index_with_multiple_templates
+
+    ActionView::PartialRenderer.expects(:collection_without_template).never
```

Comment:
Reviewing this test I don't know what this means. Why I should expect this method to never be called?